### PR TITLE
Login database

### PR DESCRIPTION
**Pull request by tetsuroescudero**
issue #23: login page now checks if email and password exist within database, if not returns false

### CODE REVIEW

File: client/src/components/Employees/Employees.js [hunk 1]
```diff
@@ -33,7 +33,6 @@ function AddEmployee() {
   };
 
   function submitClick() {
-    console.log("In click function");
```
Comment (bdeleonardis1) [inline client/src/components/Employees/Employees.js]: Let's delete this (it's not adding anything useful to the log)

File: dbtest.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const Pool = require("pg").Pool;
```
Comment (bdeleonardis1) [inline dbtest.js]: Let's delete this file it's not actually doing anything for the project. It was just to test/demonstrate the db connection.

File: server/dbtest.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const Pool = require("pg").Pool;
```
Comment (bdeleonardis1) [inline server/dbtest.js]: Let's delete this file it's not actually doing anything for the project. It was just to test/demonstrate the db connection.

File: server/routes/employees.js [hunk 4]
```diff
@@ -1,19 +1,35 @@
 var express = require("express");
```
Comment (bdeleonardis1) [inline server/routes/employees.js]: Wait what's going on here? This pr is only supposed to be for the login stuff, why is the saving employees stuff in here too? Did you branch off your saveemployees branch for this?

File: server/routes/employees.js [hunk 5]
```diff
@@ -1,19 +1,35 @@
...
+
 router.post("/employees", function(req, res) {
   var firstName = req.body.firstName;
   var lastName = req.body.lastName;
+  var email = req.body.email;
   console.log("firstName", firstName);
   console.log("lastName", lastName);
-  console.log("working");
+  console.log("email", email);
   res.json({ Successful: true });
```
Comment (bdeleonardis1) [inline server/routes/employees.js]: The convention for json keys is that they should be camelCase so let's make all of our successful's lowercase

File: client/src/components/Employees/Employees.js (hunk 1 above)
Comment (tetsuroescudero) [reply client/src/components/Employees/Employees.js]: done

File: dbtest.js (hunk 2 above)
Comment (tetsuroescudero) [reply dbtest.js]: done

File: server/dbtest.js (hunk 3 above)
Comment (tetsuroescudero) [reply server/dbtest.js]: done

File: server/routes/employees.js (hunk 5 above)
Comment (tetsuroescudero) [reply server/routes/employees.js]: done

**Comment by tetsuroescudero** [comment]
RFAL!

**Pull request merged**